node/genesis.toml

```diff
@@ -0,0 +1,21 @@
+# This is an example genesis input file for the Miden node.
+
+version = 1
+timestamp = 1672531200
+
+[[accounts]]
+type = "BasicWallet"
+mode = "RegularAccountImmutableCode"
```

Comment:
Let's get rid of the `mode` here and for `BasicFungibleFaucet` as I mentioned in the previous review.